qualtran/bloqs/basic_gates/power_test.py

```diff
@@ -87,5 +87,6 @@ def to_cirq_circuit(bloq: GateWithRegisters) -> cirq.Circuit:
     )
 
 
+@pytest.mark.slow
```

Comment:
these test_no_circular_import tests are all redundant with each other. I added it following a bug report where there was a tricky circular import that didn't usually get triggered; but I added one test because these were all in one module. During some reorganization work, the one module got split; and these tests got duplicated. Can you remove all but one of them (and mark as slow)

Reply:
I kept the ones in `qualtran.bloqs.basic_gates.power` and `qualtran.bloqs.bookkeeping.arbitrary_clifford`.

If those two are still redundant I can remove one of them.

Reply:
thank you